Maple2.Server.Game/Manager/Config/ConfigManager.cs

```diff
@@ -81,6 +84,14 @@ public class ConfigManager {
         }
     }
 
+    public void LoadChatStickers(IDictionary<int, long> stickerSets) {
```

Comment:
You don't need to pass in `stickerSets`, you can retrieve it from `session.Player.Value.Unlock.StickerSets`.